.travis.yml

```diff
@@ -11,19 +11,19 @@ services:
   - docker
 env:
   global:
-    - REPO_NAME=pyoptsparse
+    - REPO_NAME=$(eval basename $TRAVIS_REPO_SLUG)
     - DOCKER_WORKING_DIR=/home/mdolabuser/packages/$REPO_NAME
     - DOCKER_MOUNT_DIR=/home/mdolabuser/travis/$REPO_NAME
-    - SNOPT_DIR=$DOCKER_WORKING_DIR/pyoptsparse/pySNOPT/source
-    - NLPQLP_DIR=$DOCKER_WORKING_DIR/pyoptsparse/pyNLPQLP/source
+    - SNOPT_DIR=$DOCKER_WORKING_DIR/$REPO_NAME/pySNOPT/source
+    - NLPQLP_DIR=$DOCKER_WORKING_DIR/$REPO_NAME/pyNLPQLP/source
   jobs:
-    # - DOCKER_TAG=c7-gcc-ompi-py3-latest
-    - DOCKER_TAG=c7-intel-impi-py3-latest
-    - DOCKER_TAG=u18-gcc-ompi-py2-stable
-    - DOCKER_TAG=u18-gcc-ompi-py3-latest
-    - DOCKER_TAG=u18-gcc-ompi-py3-stable
-    - DOCKER_TAG=u20-gcc-ompi-py3-latest
-    - DOCKER_TAG=u20-gcc-ompi-py3-stable
+    # - DOCKER_TAG=c7-gcc-ompi-latest
+    - DOCKER_TAG=c7-intel-impi-latest
+    - DOCKER_TAG=u18-gcc-ompi-stable
+    - DOCKER_TAG=u18-gcc-ompi-latest
+    - DOCKER_TAG=u18-gcc-ompi-stable
```

Comment:
Since we removed the py2 test, I feel we have a duplicate `u18-gcc-ompi-stable` build here

Reply:
Yes very good catch.